src/solvers/refinement/string_refinement.cpp

```diff
@@ -33,42 +33,46 @@ Author: Alberto Griggio, alberto.griggio@gmail.com
 #include <langapi/language_util.h>
 #include <java_bytecode/java_types.h>
 
-string_refinementt::string_refinementt(
-  const namespacet &_ns,
-  propt &_prop,
-  unsigned refinement_bound):
-  supert(_ns, _prop),
-  use_counter_example(false),
-  do_concretizing(false),
-  initial_loop_bound(refinement_bound),
-  generator(_ns),
-  non_empty_string(false)
-{ }
-
-/// Add constraints on the size of strings used in the program.
-/// \param i: maximum length which is allowed for strings.
-/// by default the strings length has no other limit
-/// than the maximal integer according to the type of their
-/// length, for instance 2^31-1 for Java.
-void string_refinementt::set_max_string_length(size_t i)
+static bool validate(const string_refinementt::infot &info)
```

Comment:
Add some documentation. Also I don't know if it makes sense to return a `bool` if it is always true (or doesn't return)

Reply:
Has to return something so it can be called while calling another constructor. Don't think documenting is necessary (all parameters are already described in the header file).